### ISSUE TITLE
Frozen trying to translate from google

### ISSUE THREAD

**Issue by sianabanana** [ivonederlof/xlf-translator#2]
I have used your tool to create translations and all worked correctly the first time 

I then tweaked some code, deleted all previous translations and folders and went to try again.

Unfortunately, it gets to the bit where it should go to google translate and freezes

I see no network traffic or info in the console window

Am i missing something?

**Comment by abdunnahid** [comment]
Facing the same issue. Any update?

**Comment by kevplx** [comment]
@ivonederlof I am facing same issue too. Any ideas?

**Comment by rsivan** [comment]
I just forked the code and started some debugging myself, as this tool has great potential if we manage to make it work.
The first thing I noticed is there is incomplete handling of errors from the Google server.
For example, the app receives lots of BAD_REQUEST replies, probably because of the limits of the 'free' license terms.
My plan now: add improved error handling, then check the google terms and see if I need to subscribe to the service...

**Comment by kevplx** [comment]
You are right. This does have really great potential. I had to go the long
route of using a combination of transifex.com which integrates aws or azure
translate to automatically do it. But transifex isn't free and needs some
work to setup.
~ Kevin
